README.md

```diff
@@ -96,11 +96,15 @@ functionality to an existing Go application. For example:
 
     import (
             "log"
+            "time"
+            "net"
+            "os"
 
             "github.com/ibm-security-innovation/crosscoap"
     )
 
     func main() {
+            timeout := time.Duration(10)
```

Comment:
This creates a duration of 10 nanoseconds.

```suggestion
            timeout := 10 * time.Second
```

Reply:
This should be resolved now